agent/run.py

```diff
@@ -4,7 +4,8 @@
 from typing import List, Dict
 from fastapi import WebSocket
 from config import check_config_setup
-from agent.research_agent import ResearchAgent
+
+from gptresearcher.gpt_researcher import GPTResearcher
```

Comment:
@gregdrizz please add "_" to "gptresearcher" -> "gpt_researcher". Also please change the latter to "agent" under the directory instead of "gpt_researcher".